crutest/src/main.rs

```diff
@@ -106,6 +106,11 @@ enum Workload {
         #[clap(flatten)]
         cfg: RandReadWriteWorkload,
     },
+    /// Send many random reads, then report how long a single write takes
```

Comment:
It's the other way, right?  Do a bunch of writes then a single read?

Reply:
Fixed in 7465eb040360